third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -333,12 +345,20 @@ func resourceSqlDatabaseInstance() *schema.Resource {
 				ForceNew:  true,
 				Sensitive: true,
 			},
+			"source_host_port": {
```

Comment:
If GCP adds an additional field to the OnPremisesConfiguration block, then we're in an awkward situation in the provider by having broken this one out into its own field. Making this match the API by putting it in a block (in TF schema it would be a TypeList with MaxItems 1) is more future-proof.

Reply:
Changed it to the block